pycbc/workflow/core.py

```diff
@@ -1167,7 +1167,7 @@ def __init__(self, ifos, exe_name, segs, file_url=None,
         elif isinstance(segs, (segments.segmentlist)):
             self.segment_list = segs
         else:
-            err = "segs input must be either ligo.segments.segment or "
+            err = "segs input must be either igwn_segments.segment or "
             err += "segments.segmentlist. Got %s." %(str(type(segs)),)
```

Comment:
I think the second line should be changed as well to `igwn_segments.segmentlist`.

Reply:
This change has been added.